scripts/install_script.sh

```diff
@@ -0,0 +1,228 @@
+
+sudo systemctl daemon-reload
+
+sudo systemctl enable node_exporter.service
+
+sudo systemctl start node_exporter.service
+
+echo "---- Cleaning .dep .tar.gz files of grafana, prometheus, influxdb and node exporter --------"
+
+rm influxdb_1.8.3_amd64.deb grafana_7.3.1_amd64.deb node_exporter-0.18.1.linux-amd64.tar.gz prometheus-2.22.1.linux-amd64.tar.gz
+
+echo "------------Creating databases matic-------------"
+
+curl "http://localhost:8086/query" --data-urlencode "q=CREATE DATABASE matic"
+
+
+echo "--------- Cloning matic-validator-mission-control -----------"
+
+cd $HOME
+
+git clone https://github.com/vitwit/matic-jagar.git
+
+cd matic-jagar
+
+git fetch && git checkout mumbai-testnet
+
+mkdir -p  ~/.matic-jagar/config/
+
+cp example.config.toml ~/.matic-jagar/config/config.toml
+
+cd $HOME
+
+echo "------ Updatig config fields with exported values -------"
+
+sed -i '/eth_rpc_endpoint =/c\eth_rpc_endpoint = "'"$ETH_RPC_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/bor_rpc_end_point =/c\bor_rpc_end_point = "'"$BOR_RPC_ENDPOINT"'"' ~/.matic-jagar/config/config.toml
+
+sed -i '/bor_external_rpc =/c\bor_external_rpc = "'"$BOR_EXTERNAL_RPC"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_rpc_endpoint =/c\heimdall_rpc_endpoint = "'"$HEIMDALL_RPC_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_lcd_endpoint =/c\heimdall_lcd_endpoint = "'"$HEIMDALL_LCD_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_external_rpc =/c\heimdall_external_rpc = "'"$HEIMDALL_EXTERNAL_RPC"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/validator_hex_addr =/c\validator_hex_addr = "'"$VAL_HEX_ADDRESS"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/signer_address =/c\signer_address = "'"$SIGNER_ADDRESS"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/validator_name =/c\validator_name = "'"$VALIDATOR_NAME"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/stake_manager_contract =/c\stake_manager_contract = "'"$STAKE_MANAGER_CONTRACT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/tg_chat_id =/c\tg_chat_id = "'"$TELEGRAM_CHAT_ID"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/tg_bot_token =/c\tg_bot_token = "'"$TELEGRAM_BOT_TOKEN"'"'  ~/.matic-jagar/config/config.toml
+
+echo "------ Building and running the code --------"
+
+cd matic-jagar
+
+go build -o matic-jagar
+mv matic-jagar $HOME/go/bin
+
+# TODO take rpc, lcd endpoints via flags for the script and update ~/.matic-jagar/config/config.toml using sed command
+
+echo "---------- Setup Matic-Jagar service -----------"
+
+echo "[Unit]
+Description=Matic-Jagar
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=$HOME/go/bin/matic-jagar
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target" | sudo tee "/lib/systemd/system/matic_jagar.service"
+
+sudo systemctl daemon-reload
+
+sudo systemctl enable matic_jagar.service
+
+sudo systemctl start matic_jagar.service
```

Comment:
@anilCSE @PrathyushaLakkireddy It would be better to split the script in two parts. One script for installing the prerequisites which will take 2 arguments of sentry ips which are required in `prometheus.yml`. Second script for setting up the tool which takes arguments for endpoints and address. Keeping just one script will make it difficult to keep track of arguments fo the user